contracts/interfaces/modules/royalty/policies/IIpRoyaltyVault.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.23;
+
+/// @title Ip royalty vault interface
+interface IIpRoyaltyVault {
+    /// @notice Event emitted when a claim is made
+    /// @param claimerIpId The claimer ipId address
+    event Claimed(address claimerIpId);
```

Comment:
do you want to include some more details about claimed event like amount?

Reply:
Addressed in https://github.com/storyprotocol/protocol-core-v1/pull/26/commits/2def6f69f5e0429c8f3e322943bb1e781f0ea66c